prow_config.yaml

```diff
@@ -38,6 +38,7 @@ workflows:
     - test/workflows/*
     - manifests/v1alpha1/*
     - scripts/v1alpha1/*
+    - Gopkg*
```

Comment:
can you also add it into postsubmit? since the new images maybe use new dep, which need be published to public registry

Reply:
Make sense.

Reply:
Addressed, PTAL